wix-data/wix-data/guides/Error Codes.md

```diff
@@ -33,18 +33,28 @@ When you use the Wix Data API, you may encounter errors. This list includes the
 |WDE0092| Dataset is too large to sort.| The data you tried to sort didn't fit into memory. This usually occurs when large items are being sorted or there are too many items being sorted to fit into memory at once. [Add an index](https://support.wix.com/en/article/content-manager-adding-an-index-for-your-collection) to your collection to avoid using up memory.
 |WDE0093| Invalid filter for _publishStatus field, only .eq and .ne filters are allowed with 'DRAFT' and 'PUBLISHED' possible values.| When you enable the [Draft and Publish Items](https://support.wix.com/en/article/content-manager-using-the-draft-publish-items-feature) feature for a collection, a `_publishStatus` field is added to every item in that collection. If you want to filter a data request based on this field, you can only use the [`eq()`](/wix-data/wixdatafilter/eq) and [`ne()`](/wix-data/wixdatafilter/ne) functions. The only valid values for `_publishedStatus` are `'DRAFT'` and `'PUBLISHED'`. Check that your code isn't using other filter functions or values with `_publishedStatus`.
+|WDE0110| WD_BAD_REQUEST. | WixCodeNotEnabled - wix-data fails to resolve wix-code context. The error is not expected under normal circumstances, which usually means something is wrong with the system.
+|WDE0111| User request validation error. | `GridAppId` parameter must be provided in the SANDBOX segment.
 |WDE0115| WD_UNKNOWN_ERROR| Your external database requested an unsupported data operation.
 |WDE0116| WD_UNKNOWN_ERROR| Your external database responded to Wix with an error.
+|WDE0120| WD_BAD_REQUEST| Unsupported Driver Operation - `$collectionName` does not support `$operation` operation.
 |WDE0121| Sorting by multiple array fields is not supported| If there are multiple fields in your collection that contain arrays, you can't sort by both of them at once.<br><br>For example, consider a collection item that includes a field called `A` whose value is `[1,2,3]` and a field called `B` whose value is `[3,2,1]`. In this case, attempting to sort by both `A` and `B` at once results in an error. If your collection contains multiple fields that are arrays, check that you aren't trying to sort using more than one at once.
 |WDE0122| Processing time quota exceeded.| Wix limits the number of read and write requests you can make from your site per minute. The maximum number for read requests is 3000 and the maximum for write requests is 1500. We can increase these quotas in certain cases. [Learn More](https://support.wix.com/en/article/velo-about-data-quotas#overall-processing-time) about this quota and what you can do to resolve this issue.
+|WDE0123| WD_ITEM_ALREADY_EXISTS| Unique index conflict error. Possible duplicate value.
+|WDE0128| Adapter `ItemId` is missing. | Item must have an `_id` field. 
+|WDE0133| Document is too large to index.| 
+|WDE0134| Dollar($) prefix field names can not be written to the database. | Remove dollar($) prefix from the request.
```

Comment:
What context is this happening in? Is it for external DBs? I think it might be useful to mention that in the troubleshooting description.

```suggestion
|WDE0134| Dollar($) prefix field names can not be written to the database. | Make sure that none of your field names in your request have a `$` prefix.
```